frontend/src/util/index.tsx

```diff
@@ -0,0 +1,18 @@
+import moment from 'moment';
+import React from 'react';
+
+const format_date = (
+  date?: string | Date,
+  default_format?: string,
+  format?: string
```

Comment:
you can actually do `format = 'YYYY-MM-DD'` here instead to avoid creating a new variable. this will set format to 'YYYY-MM-DD' if nothing is provided